tests/console/verify_bcache_partitioning.pm

```diff
@@ -0,0 +1,57 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Validate bcache in writethrough
+# The backing device is /home and it is attached to the caching device which is the second
+# disk. Ususally the caching device is a fast device (ssd). In this scenario a rotational device is used.
+# This doesnt give any significant performance advantages but it is better to test bcache with this setup
+# than use bcache without caching(which yast provides a option if we want to do so)
+# Scenarios covered:
+# - Verify that certain values are set correctly in a bcache setup after installation;
+# - Write operation in a backing device can be performed and validate that cache is working watching the cache hits;
+# Maintainer: yiannis bonatakis <ybonatakis@suse.com>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use scheduler 'get_test_suite_data';
+use Test::Assert 'assert_true';
+
+sub run {
+
+    my $test_data = get_test_suite_data();
+
+    select_console 'root-console';
+    # assert the registered backing dev
+    assert_script_run "cat /sys/block/bcache0/bcache/backing_dev_name | grep vda4";
+    # assert bcache is running
+    assert_script_run "cat /sys/block/bcache0/bcache/running | grep 1";
+    # assert bcache is setup and is not cached dirty data
+    assert_script_run "cat /sys/block/bcache0/bcache/state | grep clean";
+
+    assert_script_run "lsblk";
```

Comment:
what about instead of just displaying partitioning, reuse module `validate_file_system` and schedule it before this one?

Reply:
let me take a look on this.

Reply:
f35918e

Reply:
that module is definitely work in progress, but at least I see that is good that we are reusing it somehow, would you mind to include in that module this line that you just delete it `assert_script_run "lsblk"` because now it is not visible [this](http://aquarius.suse.cz/tests/1910#step/verify_bcache_partitioning/14) which is really helpful, right?
Another thing is if you want to check the whole partitioning, not just the bcache part as everything is connected in this setup, but the module probably would not allow you to do that, so unless you see it easy to do now, just leave it as it is and we can improve in future.

Reply:
added

Reply:
I was referring to add it in `console/validate_file_system` but it is ok.